scripts/update-version.js

```diff
@@ -0,0 +1,15 @@
+const path = require('path');
+const fs = require('fs').promises;
+
+module.exports = async function updateCoreVersion(newVer) {
+  const filePath = `${path.resolve(__dirname, '../packages/core')}/.env`;
+
+  try {
+    const content = fs.readFileSync(filePath).toString();
```

Comment:
Shouldn't this be async too?
```suggestion
    const content = await fs.readFile(filePath).toString()